src/redis_lock/__init__.py

```diff
@@ -88,18 +92,21 @@ def id(self):
     def get_owner_id(self):
         return self._client.get(self._name)
 
-    def acquire(self, blocking=True):
+    def acquire(self, blocking=True, timeout=None):
```

Comment:
There should be a check here: if blocking=False and timeout!=None then an exception should be raised. The user is misusing the API so he must get an error to avoid subtle bugs and confusion.